docs/layouts/shortcodes/ebnf.html

```diff
@@ -0,0 +1,66 @@
+{{ $statements :=  .Inner }}
+{{ $type := default "tabs" (.Get "type") }}
+{{ $statements = replaceRE " *- *" "" $statements}}
+{{ $statements = replaceRE "\n+" "," $statements}}
+{{ $statements = replaceRE " +" "" $statements}}
```

Comment:
```suggestion
{{ $statements = replaceRE "\s+" "" $statements}}
```
doesn't this make first case redundant?

Reply:
fixed.